dart/lib/src/sentry_options.dart

```diff
@@ -47,9 +46,97 @@ class SentryOptions {
 
   int maxBreadcrumbs;
 
-  final Logger _logger;
+  /// Logger interface to log useful debugging information if debug is enabled
+  Logger _logger = noOpLogger;
+
+  Logger get logger => _logger;
+
+  set logger(Logger logger) {
+    _logger = logger != null ? DiagnosticLogger(logger, this) : noOpLogger;
+  }
+
+  /// Are callbacks that run for every event. They can either return a new event which in most cases
+  /// means just adding data OR return null in case the event will be dropped and not sent.
+  final List<EventProcessor> _eventProcessors = [];
+
+  List<EventProcessor> get eventProcessors =>
+      List.unmodifiable(_eventProcessors);
+
+  /// Code that provides middlewares, bindings or hooks into certain frameworks or environments,
+  /// along with code that inserts those bindings and activates them.
+  final List<Integration> _integrations = [];
+
+  // TODO: shutdownTimeout, flushTimeoutMillis
+  // https://api.dart.dev/stable/2.10.2/dart-io/HttpClient/close.html doesn't have a timeout param, we'd need to implement manually
+
+  List<Integration> get integrations => List.unmodifiable(_integrations);
+
+  /// Turns debug mode on or off. If debug is enabled SDK will attempt to print out useful debugging
+  /// information if something goes wrong. Default is disabled.
+  bool debug;
```

Comment:
```suggestion
  bool debug = false;
```

Reply:
but its already `false` by default, right? is it just a convention?

Reply:
no... for now Dart bools are ... null :s